pkg/provider/gitlab/gitlab.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/external-secrets/external-secrets/pkg/utils"
 )
 
-// Requires GITLAB_TOKEN and GITLAB_PROJECT_ID to be set in environment variables
+// Requires GITLAB_TOKEN, GITLAB_PROJECT_ID and GITLAB_ENVIRONMENT to be set in environment variables
```

Comment:
I don't think this is true, this line can be removed.

Reply:
Yeah mere extended the comment, but I guess you are right.